src/redux/reducers/ordersSlice.ts

```diff
@@ -12,6 +12,11 @@ export const ordersSlice = createSlice({
   name: 'orders',
   initialState,
   reducers: {
+    ordersLoadSuccess(state: IOrdersState, action: PayloadAction<IOrder[]>) {
+      state.isLoading = false
+      state.error = ''
```

Comment:
I assume that you want to set default value for error but empty string is wrong value, think about another value